src/components/NavBar/NavigationBar.tsx

```diff
@@ -1,35 +1,51 @@
 import React, {useState, useEffect} from 'react';
-import './NavigationBar.css';
-import cehatLogo from './CEHATLogo.png';
+import {Flex} from '@chakra-ui/react';
+import {MenuToggle, Logo, MenuLinks} from './Util';
```

Comment:
If possible, let's use an SVG instead of a png? I think an svg exists somewhere already

Reply:
Do you know where it exists, I can only find pngs.

Reply:
You're right, the svg was the react logo (how did that survive this long lmao), if the liaisons like the "logo" I made, I'll re-make it into a svg